libs/gi-formula-ui/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
What is the purpose of this library? As far as I understand, there won't be any apps that share this? Or is it just so the pathing for imports looks neater?

Reply:
its actually used by go-next since its using the pando UI. I assumed if we are still supporting gi-frontend it would use this as well.

Reply:
isn't go-next replacing gi-frontend?

Reply:
yeah, basically the intention is for this lib to host UI stuff that uses gi-formula,